code/game/machinery/firealarm.dm

```diff
@@ -78,9 +78,6 @@
 			src.alarm()			// added check of detector status here
 	return
 
-/obj/machinery/firealarm/attack_ai(mob/user as mob)
-	return src.attack_hand(user)
```

Comment:
This looks like a undocumented change preventing AIs from interacting with the firealarm.

Reply:
already calls attack_hand in the machinery level
![image](https://user-images.githubusercontent.com/22774890/98998228-7a00b980-253e-11eb-8b4f-4031525142bf.png)